advocacy_docs/supported-open-source/barman/pg-backup-api/03-tasks.mdx

```diff
@@ -20,68 +20,222 @@ tags:
+
+Request pg-backup-api to create instance level operations.
 
-This method is one of the most important ones, because it is where you ask pg-backup-api to create operations. Pay attention on the response below because we will use the ID to query its status later.
+Currently pg-backup-api supports only the `config_update` operation.
+
+!!! Note
+    You need to set the `content-type` header as `application/json` when executing `POST` requests to this endpoint, otherwise you will receive a `400 Bad Request error`.
 
-You need to send instructions about the operation you want to be created. This step is done by a json message (payload) which should look like this:
+##### config_update
 
-The first supported (and only for the moment) operation type is "recover".
+This operation is used to request a remote execution of `barman config-update` command.
 
+The JSON payload of the request must contain the following keys:
+
+ * `type`: set as `config_update` to request that operation
+ * `changes`: an array of documents to be used as argument for `barman config-update`
+
+As an example, you could request pg-backup-api to update the configuration of a couple Barman servers plus a Barman model with a payload like this:
+
+```json
+{
+  "type": "config_update",
+  "changes": [
+    {
+      "scope": "server",
+      "server_name": "my-server-a",
+      "archiver": "on",
+      "streaming_archiver": "off"
+    },
+    {
+      "scope": "server",
+      "server_name": "my-other-server-b",
+      "conninfo": "some_conn_info",
+      "streaming_conninfo": "some_streaming_conninfo",
+      "streaming_archiver": "on"
+    },
+    {
+      "scope": "model",
+      "model_name": "some-model",
+      "streaming_conninfo": "some_streaming_conn_info"
+    }
+  ]
+}
+```
+
+Assume the above JSON content is stored in a file named `payload-pg-backup-api.json`, you could issue a request to the API like this:
+
+```bash
+curl -X POST http://barman-server.my.org/operations -H "content-type: application/json" -d@payload-pg-backup-api.json
+{
+  "operation_id": "20240118T104231"
+}
 ```
-{"operation_type": "recover",
- "backup_id": "20230221T155931",
- "remote_ssh_command": "ssh postgres@db_server_two.my.org",
- "destination_directory": "/var/lib/pgdata"}
+
+The `operation_id` in the response is the ID of the operation that has been created in by the API.
+
+#### GET: /operations/OPERATION_ID
+
+This endpoint allows you to check the status of an instance level operation with ID `OPERATION_ID`, which can be either of these:
+
+ * `DONE`: if the operation finished successfully
+ * `FAILED`: if the operation finished with errors
+ * `IN_PROGRESS`: if the operation is still ongoing
+
+ Please find an example below:
+
+ ```bash
+ curl http://barman-server.my.org/operations/20240118T104231
+ {
+  "operation_id": "20240118T104231",
+  "status": "DONE"
+}
+ ```
+
+#### GET: /servers/NAME/operations
+
+Returns all server level tasks created by the API for Barman server `NAME`, if any.
```

Comment:
Add below:
The following example shows the two tasks created on the `my-barman-server` Postgres server.